doc/parameters/param_list.md

```diff
@@ -175,7 +175,18 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | `<filter name>`.enabled | true | Whether it is enabled |
-| `<filter name>`.filter_info_topic | N/A | Name of the CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.filter_info_topic | N/A | Name of the incoming CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.transform_tolerance | 0.0 | TF tolerance |
```

Comment:
This should not be 0.0, that's not very robust. Make this `0.1` and update in the code

Reply:
Updated in https://github.com/ros-planning/navigation2/pull/2074/commits/2223741f3753bc368f1e5774a8a8f0c6d66a15d6